initializer_inputs/agriculture/agriculture_burner_network_gas_share.ad

```diff
@@ -0,0 +1,4 @@
+- id = 246
```

Comment:
See [this comment](https://github.com/quintel/etsource/issues/1227#issuecomment-276033822)